californium-core/src/test/java/org/eclipse/californium/core/test/lockstep/LockstepEndpoint.java

```diff
@@ -722,6 +725,49 @@ public ResponseExpectation loadMID(final String var) {
 			super.loadMID(var);
 			return this;
 		}
+		
+		public ResponseExpectation newMID(final String var) {
```

Comment:
Should I really "break" the javadoc practice used by this class :-)?

(I will add a javadoc.)